content/_layouts/simplepage.html.haml

```diff
@@ -10,10 +10,8 @@ layout: default
 - unless page.noanchors
   = partial("anchors.html.haml")
 
-.container
-  .row.body
-    .col-md-12
-      - unless page.notitle
-        %h1
-          = page.title
-      = content
+.app-container
```

Comment:
This will impact more pages than just the one relating to this PR - largely just adds some more padding to the pages but reduces reliance on bootstrap.

Reply:
@janfaracik are there any pages you have run into as an example? I was checking out the deploy site but didn't come across anything especially obvious.

Reply:
https://www.jenkins.io/chat/ is a simple page that now looks like crap.